en/identity-server/next/docs/guides/authentication/oidc/implement-login-with-par.md

```diff
@@ -0,0 +1,4 @@
+{% set product_name = "WSO2 Identity Server" %}
```

Comment:
We don't need to assign this in every page now.

Reply:
Removed with 3afb52015e9bdb7c24a69552d5749401f7fdba04